src-self-hosted/Module.zig

```diff
@@ -2081,12 +2081,15 @@ fn createNewDecl(
 }
 
 /// Get error value for error tag `name`.
-pub fn getErrorValue(self: *Module, name: []const u8) !u16 {
+pub fn getErrorValue(self: *Module, name: []const u8) !std.StringHashMapUnmanaged(u16).Entry {
```

Comment:
This looks like it can be optimized following the same pattern as the other review comment